packages/components/src/calendar/utils/use-controlled-value.ts

```diff
@@ -1,7 +1,10 @@
 /**
  * WordPress dependencies
  */
-import { useState, useCallback } from 'react';
+/**
+ * External dependencies
+ */
+import { useState, useCallback } from '@wordpress/element';
```

Comment:
Probably an auto-suggest hiccup

```suggestion
import { useState, useCallback } from '@wordpress/element';
```